src/WooCommerce.php

```diff
@@ -679,7 +679,7 @@ public static function woocommerce_after_checkout_billing_form() {
     if (!$is_subscriber) {
       $name = is_checkout() ? 'woocommerce_checkout_place_order' : 'subscriber_associate_submit';
       echo '<p id="subscriber_submit_field" class="form-row form-actions" data-priority="-20">';
-      echo '<button id="subscriber_associate_submit" type="submit" name="' . $name . '" value="1" class="button button--primary">';
+      echo '<button id="subscriber_associate_submit" type="submit" name="' . $name . '" value="1" class="button">';
```

Comment:
However, this is the only/primary button in the subscription association form on https://test.stimme.de/shop/user/subscriptions

Can we only remove the primary class in the checkout?

Reply:
Sorry @sun but I don't get what you mean.

I just opened the page you linked and the only button I see is the back button (`Weitere Angebote`). Nothing with the `#subscriber_associate_submit` found there.

How this code is connected to that page…? 🤔

Reply:
The subscriber association form only appears when the user account is not associated with a subscription yet (after enabling the checkbox).   You'd have to register with a new test account to see it; e.g., luca+stim1@netzstrategen.com